lib/arrays.md

```diff
@@ -5,44 +5,60 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is an Array?
 
-Replace me with your response...
+An Array is a data object that stores an ordered list of values.
 
 # What are some examples of information that would be Arrays as opposed to some other data type?
 
-Replace me with your response...
+a collection of elements, each selected by one ore more identifying keys. 
```

Comment:
Below, when you call `arr[5]`, are you selecting an element from the Array by a **key** or some other identifying piece of information? 

In Ruby, what kind of lists _do_ use keys?